HotelBooking.Domain/Room.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace HotelBooking.Domain;
+
+/// <summary>
+/// Инофрмация о номере
+/// </summary>
+public class Room 
+{
+    /// <summary>
+    /// Идентификатор номера
+    /// </summary>
+    public required int Id { get; set; }
+
+    /// <summary>
+    /// Тип номера
+    /// </summary>
+    public required RoomType Type { get; set; }
+
+    /// <summary>
+    /// Вместимость
+    /// </summary>
+    public required int Capacity { get; set; }
+
+    /// <summary>
+    /// Цена
+    /// </summary>
+    public required int Decimal { get; set; }
```

Comment:
> Цена должна быть `decimal`

Я начинаю терять покой в сердце
```suggestion
    /// <summary>
    /// Цена
    /// </summary>
    public required decimal Cost { get; set; }
```